### PR TITLE
Use window options

### PR DESCRIPTION
**Pull request by obany**
Example to demonstrate the use of the window options

### CODE REVIEW

**Review by johnman — APPROVED** [review]

File: how-to/use-window-options/public/manifest.fin.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "runtime": {
+    "arguments": "--v=1 --inspect",
+    "version": "stable"
```
Comment (adam-saland) [inline how-to/use-window-options/public/manifest.fin.json:4]: Do we want to put a runtime version in here? I forget how we wanted to approach...

**Review by adam-saland — CHANGES_REQUESTED** [review]
Awesome sauce.

File: how-to/use-window-options/public/manifest.fin.json (hunk 1 above)
Comment (obany) [reply how-to/use-window-options/public/manifest.fin.json:4]: All the other examples have stable, they should work with most recent versions. Once we have the GH automated builds we will see if stuff breaks

Comment (johnman) [reply how-to/use-window-options/public/manifest.fin.json:4]: lets get this in but then we will do a pass that will pin to the next runtime release (102). And I think maybe we should pick a port as some are 8080 and I think others are 5050. Maybe 5050 so you can run a container sample alongside a workspace starter sample?

**Pull request merged**